wdl-lint/tests/lints/missing-meta-and-parameter_meta/source.wdl

```diff
@@ -0,0 +1,15 @@
+## This is a test of missing both the meta and parameter_meta
+
+version 1.0
+
+workflow test {
+    input {
+        File input_file
+    }
+    call test_task {
+        input: input_file = input_file
+    }
+    output {
+        File output_file = test_task.output_file
+    }
+}
```

Comment:
We should remove the missing newline diagnostic in `source.errors` with a newline here.